openshift/e2e-tests.sh

```diff
@@ -22,7 +22,7 @@ failed=0
 
 # (( !failed )) && install_tracing || failed=1
 
-(( !failed )) && run_e2e_tests || failed=1
+# (( !failed )) && run_e2e_tests || failed=1
 
 (( !failed )) && run_conformance_tests || failed=1
 
```

Comment:
of course - this will change - after tests and moving forward with the new "goal"